xarray/tests/test_units.py

```diff
@@ -8,6 +8,9 @@
 from xarray.core.npcompat import IS_NEP18_ACTIVE
 
 pint = pytest.importorskip("pint")
+DimensionalityError = pint.errors.DimensionalityError
```

Comment:
This will raise if pint isn't installed, can we put it [here](https://github.com/pydata/xarray/pull/3238/files/bc85c3b64ace6d5ba8d8ff8dce2f5406953f3e47..4b4605d05bffeb2d4d449e2be4c87c50786e769a#diff-45289dbef3724b51d0b667552f57592cR33) ?

Reply:
Won't raise because of the line just before it
``python
pint = pytest.importorskip("pint")
```

Reply:
Mea culpa. I had thought that sent `pint` to `None`, but it skips the whole module.